ProcessLib/SmallDeformation/SmallDeformationProcessData.h

```diff
@@ -25,12 +25,12 @@ struct SmallDeformationProcessData
     SmallDeformationProcessData(
         std::unique_ptr<MaterialLib::Solids::MechanicsBase<DisplacementDim>>&&
             material)
-        : _material{std::move(material)}
+        : material{std::move(material)}
```

Comment:
This seems to be not a problem, although I would expect that there is a problem with the same names. Just for curiosity: Can someone explain why this is not a problem?